webservices/rest.py

```diff
@@ -215,6 +216,8 @@ def add_aggregate_resource(api, view, schedule, label):
 
 api.add_resource(download.DownloadView, '/download/<path:path>/')
 
+api.add_resource(legal.UniversalSearch, '/legal/search/')
```

Comment:
Curious about the name `UniversalSearch`, are you thinking this will encompass more things than just legal resources?

Only ask because I'm seeing similar themes come up in the UI/web-app side so it would be good to be on the same page as the API. I don't have an opinion myself, yet.